### ISSUE TITLE
Linux version not running on Ubuntu

### ISSUE THREAD

**Issue by ShantySuffisance** [hisschemoller/JS-EPG#19]
MPG looks like great fun! I tried the mpg_2_2_installer_lin.deb on Ubuntu 18 LTS but many elements of the gui are missing and resizing the window destroys the gui even more: 

![bug_MPG](https://user-images.githubusercontent.com/63818756/149365654-ebc930aa-ea63-40fb-91a8-010bab2dc0bb.png)

the Euclidean processor looks wrong, the Euclid FX and the output are invisible. What can I try to run MPG? 
Thank you!

**Comment by hisschemoller** [comment]
Ah, sorry about the mess, thanks for the screenshot. Currently my Linux / Windows laptop is broken so I won't be able to fix the version for these platforms. Is the online version of the app broken in the same way?

**Comment by ShantySuffisance** [comment]
no, the online version of the app seems even more broken. Neither at https://www.hisschemoller.com/mpg/ nor when i start the "src/index.html" locally can i see more than the menu bar at the top. It does not react to interaction. (if it is relevant: i tried in firefox v103.0)
